src/root.component.tsx

```diff
@@ -1,14 +1,46 @@
 import React from "react";
-import { Route } from "react-router";
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientBanner from "./summary/banner/patient-banner.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { LevelTwoRoutes } from "./summary/level-two-routes";
 
 function Root(props) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+  const [showPatientSummary, setShowPatientSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient => {
+      setCurrentPatient(patient);
+      createErrorHandler();
```

Comment:
You have put `createErrorHandler()` inside of the success function. This is a bug. Here's what this should be:

```tsx
const subscription = getCurrentPatient().subscribe(
  patient => setCurrentPatient(patient),
  createErrorHandler()
)
```

Reply:
You didn't fix this.